conf/app.routes

```diff
@@ -0,0 +1,6 @@
+# microservice specific routes
+
+GET        /assets/*file                      controllers.Assets.versioned(path="/public", file: Asset)
```

Comment:
do we need `/assets/` here as its a backend service ?

Reply:
guess not, they're gone, out the door!